# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux26mips/video-board-quirk.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- linux.git/arch/mips/sgi-ip30/ip30-xtalk.c.vanilla	2005-11-12 00:42:04.000000000 +0100
+++ linux.git/arch/mips/sgi-ip30/ip30-xtalk.c	2005-11-12 00:42:44.000000000 +0100
@@ -170,5 +170,5 @@
 	}
 	i=IP30_XTALK_NUM_WID;
 	while((i=ip30_xtalk_find(PCIBR_XTALK_MFGR,PCIBR_XTALK_PART,i))!=-1)
-		bridge_probe(0, i, IP30_WIDGET_HEART);
+		if (i!=11)bridge_probe(0, i, IP30_WIDGET_HEART);
 }
